### PR TITLE
Updated Layout to look like UWP

### PR DESCRIPTION
**Pull request by albertoxamin**

### CODE REVIEW

File: UnofficialSteamAuthenticator/MainPage.xaml.cs [hunk 1]
```diff
@@ -41,6 +41,13 @@ private void WebNotify(object sender, NotifyEventArgs e)
 
         protected override void OnNavigatedTo(NavigationEventArgs e)
         {
+            UsersButton.Content = StringResourceLoader.GetString("UsersButton/Label");
+            AboutButton.Content = StringResourceLoader.GetString("AboutButton/Label");
+            Message.Text = StringResourceLoader.GetString("MessageButton/Label");
+            Confirmations.Text = StringResourceLoader.GetString("ConfirmationsButton/Label");
+            SteamGuard.Text = StringResourceLoader.GetString("SteamGuardButton/Label");
```
Comment (Top-Cat) [inline UnofficialSteamAuthenticator/MainPage.xaml.cs:48]: These Strings are named Button.Label not Button/Label

Comment (albertoxamin) [inline UnofficialSteamAuthenticator/MainPage.xaml.cs:48]: I know, but in order to access them via c# the '.' must be replaced with '/'
Proof: ( http://stackoverflow.com/questions/20747707/metro-getting-strings-from-resources-with-globalization )

**Comment by Top-Cat** [comment]
Positives
- Bigger code is better

Negatives
- Oh my the spacing
- Animation is gone

Screenshot

**Comment by Top-Cat** [comment]
Can one of the admins verify this patch?

**Comment by Top-Cat** [comment]
test this please <- ignore this, testing jenkins

**Comment by Top-Cat** [comment]
test this please <- ignore this, testing jenkins

**Comment by Top-Cat** [comment]
Oh, this actually breaks major functionality too
Screenshot

**Comment by albertoxamin** [comment]
Ok, i'll fix that

**Comment by Top-Cat** [comment]
Is there not a better way of implementing this than just trying to emulate the right look with panels and buttons?

**Comment by albertoxamin** [comment]
The best way would be creating a custom user control for the command bar.

**Comment by Top-Cat** [comment]
I more meant in a UWP project wouldn't the command bar render like that anyway
(That being the proper solution)

I think the feeling is a hamburger menu like the official app on android has would be the preferred option really. See https://blogs.msdn.microsoft.com/quick_thoughts/2015/06/01/windows-10-splitview-build-your-first-hamburger-menu/

This is not high on my priorities though, I'd like to work on tests, notifications and chat first (and none of that is small).

**Pull request closed without merging**